doc/source/rllib/doc_code/rlmodule_guide.py

```diff
@@ -156,9 +158,9 @@ def __init__(self, config: RLModuleConfig) -> None:
         super().__init__(config)
 
     def setup(self):
-        input_dim = self.config.observation_space.shape[0]
-        hidden_dim = self.config.model_config_dict["fcnet_hiddens"][0]
-        output_dim = self.config.action_space.n
+        input_dim = self.observation_space.shape[0]
```

Comment:
note: the old notation: `self.config....` still works in case users have old RLModule classes that use this attribute, but it is no longer advertized.

Reply:
Maybe we should set a date until users should switch to be able to deprecate cleanly.